invenio_app_rdm/config.py

```diff
@@ -1096,3 +1108,51 @@ def github_link_render(record):
 }
 
 STATS_PERMISSION_FACTORY = permissions_policy_lookup_factory
+
+
+# Invenio-Notifications
+# =================
+# See https://github.com/inveniosoftware/invenio-notifications/blob/master/invenio_notifications/config.py  # noqa
+
+
+NOTIFICATIONS_BACKENDS = {
+    EmailNotificationBackend.id: EmailNotificationBackend(),
+}
+"""Notification backends."""
+
+
+NOTIFICATIONS_BUILDERS = {
+    CommunityInclusionSubmittedNotificationBuilder.type: CommunityInclusionSubmittedNotificationBuilder,
+}
+"""Notification builders."""
+
+
+NOTIFICATIONS_ENTITY_RESOLVERS = [
+    RDMRecordServiceResultResolver(),
+    ServiceResultResolver(service_id="users", type_key="user"),
+    ServiceResultResolver(service_id="communities", type_key="community"),
+    ServiceResultResolver(service_id="requests", type_key="request"),
+    ServiceResultResolver(service_id="request_events", type_key="request_event"),
+]
+"""List of entity resolvers used by notification builders."""
+
+
+class UserPreferencesNotificationsSchema(UserPreferencesSchema):
```

Comment:
minor: to be moved to invenio_app_rdm/users/schemas.py, lets try to minimise the config.py  as much as possible!